verilog/sim_two_power_mod.cpp

```diff
@@ -18,35 +18,42 @@ struct RSATwoPowerModIn {
     os << "{" << v.power << ", " << v.modulus << "}" << std::endl;
     return os;
   }
-
   verilog::vuint<32> power;
   KeyType modulus;
 };
 using RSATwoPowerModOut = KeyType;
 
-const char str_modulus[] =
-    "0xE07122F2A4A9E81141ADE518A2CD7574DCB67060B005E24665EF532E0CCA73E1";
-const char str_ans[] =
-    "0AF39E1F831CB4FCD92B17F61F473735C687593A931C97D2B60AD6C7443F09FDB";
-const int power = 512;
+void KillSimulation() {
+  wait(100, SC_NS);
+  sc_stop();
+}
 
 SC_MODULE(Testbench) {
   sc_clock clk;
   DUTWrapper<DUT> dut_wrapper;
   shared_ptr<Driver<RSATwoPowerModIn>> driver;
   shared_ptr<Monitor<RSATwoPowerModOut>> monitor;
+  unique_ptr<ScoreBoard<RSATwoPowerModOut>> score_board;
 
 public:
   SC_HAS_PROCESS(Testbench);
   Testbench(const sc_module_name &name)
-      : sc_module(name), clk("clk", 1.0, SC_NS), dut_wrapper("dut_wrapper") {
+      : sc_module(name), clk("clk", 1.0, SC_NS), dut_wrapper("dut_wrapper"),
+        score_board(new ScoreBoard<RSATwoPowerModOut>(&KillSimulation)) {
```

Comment:
No need for & (while they are the same lol)

Reply:
fixed.